Copy editing (especially in ImageMosaic sections)

maddenp opened the pull request:
I have been carefully reading the ImageMosaic sections to address some issues I am having, and have noticed some typos, inconsistencies and syntax issues I thought I would try to help correct.

In many files, the only changes are to consistently capitalize "GeoServer" and "ImageMosaic". There are probably still instances of "geoserver" used where "GeoServer" should be, but these are more difficult to differentiate from legitimate uses in filenames and code examples, and I don't have time at the moment to hunt them all down.

In other places, I have made changes that I believe lead to more idiomatic English.

I have made inline comments in other places where I thought my intent might be less obvious.

maddenp commented on doc/en/user/source/data/raster/imagemosaic/configuration.rst:
```diff
@@ -73,11 +73,11 @@ The table below describes the various elements in this configuration file.
      - Suggested plugin for reading the image files.
    * - Envelope2D
-     - Y
+     - N
```
This option seems not to actually be required. In any case, it does not appear in the example configuration shown below, so either this line or the configuration is wrong. (That is, shouldn't the option appear in the example configuration if it is mandatory?)

maddenp commented on doc/en/user/source/data/raster/imagemosaic/configuration.rst:
```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
    * - AbsolutePath
-     - Y
-     - Controls whether or not the path stored inside the ``location`` attribute represents an absolute path or a path relative to the location of the shapefile index. Notice that a relative index ensures much more portability of the mosaic itself. Default value for this parameter is ``false``, which means relative paths.
+     - N
```
Similar to the previous comment, either this option is not actually required, or the example configuration shown below should include it (I think).

dvntucker replied:
AbsolutePath is *not* required.

maddenp commented on doc/en/user/source/data/raster/imagemosaic/configuration.rst:
```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
    * - CanBeEmpty
      - N
-     - Boolean flag used for configuring empty mosaics. When enabled the imagemosaic will not throw an exception caused by the absence of any coverage. By default it is set to false.
+     - Boolean flag used for configuring empty mosaics. When enabled the ImageMosaic will not throw an exception caused by the absence of any coverage. By default it is set to ``false``.
```
In a number of places, I have added double-backticks around boolean values for consistency. Mostly, the original document had them in backticks, so I tried treat all instances the same.

maddenp commented on doc/en/user/source/data/raster/imagemosaic/configuration.rst:
```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
+     - Boolean flag used for configuring empty mosaics. When enabled the ImageMosaic will not throw an exception caused by the absence of any coverage. By default it is set to ``false``.
    * - Envelope2D
-     - Y
+     - N
```
Again, apparently not required, or missing from the given example config.

maddenp commented on doc/en/user/source/data/raster/imagemosaic/tutorial.rst:
```diff
@@ -318,13 +318,13 @@ Multi-resolution imagery with reprojection
 As a general rule, we want to have the highest resolution granules shown "on top", with the lower-resolution granules filling in the gaps as necessary.
 
-In this example, we will serve up overlapping granules that have varying resolutions. In addition, we will mix resolutions, such that the higher resolution granule is reprojected to match the resolution of the 
+In this example, we will serve up overlapping granules that have varying resolutions. In addition, we will mix resolutions, such that the higher resolution granule is reprojected to match the resolution of the lower resolution granules.
```
This sentence was previously incomplete, so I have guessed here about the correct end. I hope I got it right.

tbarsballe replied:
That seems like the correct ending.

maddenp commented on doc/en/user/source/data/raster/imagemosaic/configuration.rst:
````diff
@@ -74,7 +74,7 @@ The table below describes the various elements in this configuration file.
    * - Envelope2D
      - N
-     - Envelope for the mosaic formatted as ```LLX,LLY URX,URY`` (notice the space between the lower left and upper right coordinate pairs).
+     - Envelope for the mosaic formatted as ``LLX,LLY URX,URY`` (notice the space between the lower left and upper right coordinate pairs).
````
I *think* the exta backtick is not actually part of the syntax as it should appear in the configuration file.

tbarsballe commented on doc/en/user/source/data/raster/imagemosaic/configuration.rst:
```diff
@@ -104,11 +104,11 @@ A sample configuration file follows::
 By default the ImageMosaic index is specified by a shapefile, which is located at the root of the ImageMosaic directory, just like the primary configuration file.
 
-If needed different storage can be used for the index like a spatial DBMS which is the preferred solution when willing to share the ImageMosaic itself in a cluster of GeoServer intances. In this case the user must supply GeoSerer with the proper onnection parameters which can be specified by using a :file:`datastore.properties` file placed at the root of the ImageMosaic directory.
+If needed, different storage can be used for the index -- like a spatial DBMS, which is the preferred solution when you wish to share the ImageMosaic itself in a cluster of GeoServer instances. In this case the user must supply GeoServer with the proper connection parameters, which can be specified by using a :file:`datastore.properties` file placed at the root of the ImageMosaic directory.
```
I don't think there should be two dashes (`--`) here. Were you trying to use an em-dash?

maddenp replied:
I'll fix this throughout...

tbarsballe commented on doc/en/user/source/data/raster/imagemosaic/tutorial.rst:
```diff
@@ -239,8 +239,8 @@ The result is the following.
-This mosaic, formed by two single granules, shows a typical case where the "nodata" collar areas of the granules overlap, as hown in the picture above.
-In this case we can use the ``InputTransparentColor`` parameter to make the collar areas disappear during the superimposition process, as instance, in this case, by using an ``InputTransparentColor`` of ``#FFFFFF``  
+This mosaic, formed by two single granules, shows a typical case where the "nodata" collar areas of the granules overlap, as shown in the picture above.
+In this case we can use the ``InputTransparentColor`` parameter to make the collar areas disappear during the superimposition process -- in this case, by using an ``InputTransparentColor`` of ``#FFFFFF``.
```
Should this be an em-dash (—)?

maddenp commented:
Hi Torben -- I squashed the commits, hopefully correctly. Please have a look and see if it's ok. I also changed the double-hyphens to em-dashes as suggested.

tbarsballe commented:
Looks good, thanks for all the fixes. Merging.
